rest-server/build.sbt

```diff
@@ -174,3 +173,16 @@ lazy val server = (project in file("server")).dependsOn(core, common, engines, a
     SbtExclusionRule("org.slf4j", "slf4j-log4j12")
   )
 ).enablePlugins(JavaAppPackaging).aggregate(core, common, engines, admin)
+
+def generateBuildInfo: Setting[_] =
+  sourceGenerators in Compile += Def.task {
+    val file = (sourceManaged in Compile).value / "com" / "actionml" / "admin" / "BuildInfo.scala"
```

Comment:
@andreyshikov I changed it to be in com.actionml.admin since there is no com.actionml.harness